### DISCUSSION TITLE
row-span-15 not taking effect

### DISCUSSION THREAD

**Discussion by tian-yan** [tailwindlabs/tailwindcss#4480] [Help] [answered]
I added some customization `gridTemplateRows: {20: 'repeat(20, minmax(0, 1fr))'}` in file `tailwind.config.js` aiming to produce 20 equal-height rows, to serve my ultimate goal --- sizing the height of each HTML section in proportion to screen height as layout for Mobile Screen.  
Demo code: https://play.tailwindcss.com/wENKaJKjEb
However, the `row-span-15` just did not work after re-compiling the CSS file? did I missed out anything ?

**Comment by tian-yan** [comment, marked as answer]
figured out by myself. In order to use command `row-span-15`, customization is also required. 
Adding `gridRow: {'span-7': 'span 15 / span 15',}` into `tailwind.config.js` and re-compiling CSS file fix the problem. 
The query I posted above turned out to be a silly one.. :(

**Reply by simonswiss** [reply]
That's it! The [default values for the `row-span` utilities](https://tailwindcss.com/docs/grid-row) stop at `span-6`, so you need to add your custom `span-15` value for it to work 👍 

I wouldn't worry about asking a "silly question" - this thread may help someone else in the future, so it's still definitely useful! ✨
